src/App.tsx

```diff
@@ -15,36 +15,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+export enum SortKeys {
+  ALPHABETICAL = 'Alphabetical',
+  LENGTH = 'Length',
+}
+
+export const App: FC = () => {
+  const [sort, setSort] = useState<string>('');
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  const getSortedList = (): string[] => {
+    let sortedGoods = [...goodsFromServer];
```

Comment:
```suggestion
    const sortedGoods = [...goodsFromServer];
```
Prefer const over let